src/components/EditProfile.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import { Form, Button, Container } from 'semantic-ui-react';
+
+export default class EditProfile extends Component {
```

Comment:
Since this component is not handling internal state, you can make it a functional component